python/test/unit/tools/test_aot.py

```diff
@@ -234,6 +234,8 @@ def _compile_kernel(dir, signature, kernel_name, out_name, out_path, num_warps,
             str(num_warps),
             "-g",
             grid,
+            "-t",
```

Comment:
We should add parameters to `_compile_kernel` to control this from the specific tests rather than hardcoding it here which breaks all non-Gluon and NVIDIA AOT tests?